src/App/App.js

```diff
@@ -66,6 +65,10 @@ export class App extends Component{
   }
 
   async componentDidMount() {
+    this.loadAllMovies()
```

Comment:
great called on putting this fn into a method that we could reuse throughout our code